app/releasemanager.cpp

```diff
@@ -20,7 +20,9 @@
 #include "releasemanager.h"
 #include "drivemanager.h"
 
-#include "isomd5/libcheckisomd5.h"
+extern "C" {
+#include <libcheckisomd5.h>
```

Comment:
TBH I don't see why one way would be better than the other and I'm not totally convinced either. I don't want to discuss something like that with bcl when I'm not convinced.